README.md

```diff
@@ -1,12 +1,13 @@
-![Logo](assets/logo.png)
+![Logo][]
```

Comment:
Using link-references (to improve readablity). Will fix this everywhere in the file.